mlflow/tensorflow/__init__.py

```diff
@@ -138,6 +140,32 @@ def log_model(
     """
     Log a TF2 core model (inheriting tf.Module) or a Keras model in MLflow Model format.
 
+    .. note::
```

Comment:
Do we have a follow up PR to add this note to the docs page for the TF flavor in models? This is pretty important.

Reply:
Yes. Sure.